### ISSUE TITLE
'Clear Cache' under Settings does not clear project cache

### ISSUE THREAD

**Issue by devillex** [Shippable/support#2720]
I tried clearing the cache from the project settings tab, but it failed to clear my cached node modules. What broke my build was when Angular renamed their NPM packages recently. What fixed it for me was to include the suggested [reset minion] or [reset_minion] in the commit message.

- Excerpt from documentation:

> Clearing cache

> You can clear cache in one of two ways:
> - Including [reset minion] or [reset_minion] in your commit message.
> - Clicking on the Clear cache in your Project settings UI.

- Excerpt from yml:
```yml
  cache: true

  cache_dir_list:
    - $SHIPPABLE_BUILD_DIR/node_modules
```

- Link to the build failure
https://app.shippable.com/runs/57692909cb11530c00fc2c13

**Comment by a-murphy** [comment]
I haven't been able to reproduce this with manually triggered builds or commit webhooks.  Clicking "clear" then "confirm" and triggering a build ran the build without using the cache, as expected.  Am I missing something unusual about this build?

**Comment by devillex** [comment]
I haven't had a chance to test, but was this maybe resolved by the chance for #2721?

**Comment by devillex** [comment]
I can confirm that this now works perfectly. Thanks for the support! I believe it may have been fixed with what was done to resolve #2721.
